kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -160,13 +148,31 @@ var StarView = BaseView.extend({
     },
 
     render: function() {
-        this.$el.html(this.template(this.model.attributes));
+        var template_options = {};
+        _.extend(template_options, this.model.attributes, {title: this.title, lowest_value: this.label_values[1], highest_value: this.label_values[5]});
+        this.$el.html(this.template(template_options));
         this.rating_change();
     },
 
-    rate_value_callback: function(ev) {
-        var val = $(ev.target).attr("data-val");
+    // Throttled so it's triggered only once when several children are the targets of the "click" event
+    rate_value_callback: _.throttle(function(ev) {

```

Comment:
This one is actually throttled so that clicking on a child element within the `.star-rating-option` container doesn't trigger multiple events (`trailing: false` option prevents it from being called at the end of the throttle period as well). Note the callback from the events hash:
```js
        "click .star-rating-option": "rate_value_callback",
        "click .star-rating-option >": "rate_value_callback",
```